Tests/IsIdentifiableTests/default.yaml

```diff
@@ -0,0 +1,281 @@
+
```

Comment:
We should remove the unrelated microservice options from this config

Reply:
I've left some in and clarified in a comment why (interoperability of `ii` with library use - i.e. shared config).